documentation/asciidoc/accessories/camera/raspicam.adoc

```diff
@@ -2,6 +2,34 @@
 
 `raspistill`, `raspivid` and `raspiyuv` are command line tools for using the camera module.
 
+=== Enabling the Camera
+
+Before using any of the _Raspicam_ applications, the camera must be enabled.
+
+==== On the desktop
+
+Select `Preferences` and `Raspberry Pi Configuration` from the desktop menu: a window will appear. Select the `Interfaces` tab, then click on the `enable camera` option. Click `OK`. You will need to reboot for the changes to take effect.
```

Comment:
Seems to be duplicated info in camera_installing.adoc ?

Reply:
Huh, I seem to have confused myself a bit by moving the content of camera_installing.adoc into camera_hardware.adoc but then forgetting to remove that file. I'll fix that up along with everything else!